addon/components/offering-form.js

```diff
@@ -146,6 +147,16 @@ export default class OfferingForm extends Component {
     this.learnerGroups = this.learnerGroups.filter(g => !groupsToRemove.includes(g));
   }
 
+  @action
+  async addLearner(learner) {
+    this.learners = [...this.learners, learner];
+  }
+
+  @action
+  async removeLearner(learner) {
+    this.learners = this.learners.filter(l => l !== learner);
```

Comment:
I've recently seen some performance pattern with comparing ember-data models. They're pretty deep objects with a lot of metadata. Probably better to check the ID if possible.

Reply:
good point, fixed.